src/apps/dictionaries/pages/CreateDictionaryPage.tsx

```diff
@@ -34,9 +36,13 @@ const CreateDictionaryPage: React.FC<Props> = ({
   errors,
   createSourceAndDictionary,
   loading,
+  resetCreateDictionary,
   newDictionary
 }: Props) => {
   const previouslyLoading = usePrevious(loading);
+  
+  // eslint-disable-next-line react-hooks/exhaustive-dep
```

Comment:
```suggestion
  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Linter rule names are very specific!